src/agents/controller/index.ts

```diff
@@ -96,6 +96,20 @@ Provide your analysis based on the schema requirements.
 			});
 		}
 
+		if (analysis.version.includes("-next")) {
```

Comment:
this is imperative. can you instead update the system prompt above to add a rule. something like:

`7. If the release or tag contains a -next in it, it should be ignored`